be/src/olap/tablet_meta.cpp

```diff
@@ -1209,9 +1209,10 @@ void DeleteBitmap::remove_stale_delete_bitmap_from_queue(const std::vector<std::
                 }
                 auto start_bmk = std::get<1>(delete_bitmap_tuple);
                 auto end_bmk = std::get<2>(delete_bitmap_tuple);
+                //to remove delete bitmap doesn't contains end_bmk
```

Comment:
Here to explain in detail, in the future, when others come to maintain the code here, may not be clear about the reason for the -1 operation here and ignore this detail.